src/index.html

```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <script src="https://kit.fontawesome.com/e4fb871c71.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
```

Comment:
You don't have to import the styles as it is compiled with webpack through the JavaScript

Reply:
sorry for that i just added this for locally test